src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -17,6 +17,7 @@
 from assisted_test_infra.test_infra import BaseClusterConfig, BaseInfraEnvConfig, ClusterName, exceptions, utils
 from assisted_test_infra.test_infra.controllers.load_balancer_controller import LoadBalancerController
 from assisted_test_infra.test_infra.controllers.node_controllers import Node
+from assisted_test_infra.test_infra.controllers.node_controllers.libvirt_controller import LibvirtController
```

Comment:
No need to import specific controller in `cluster.py`. See my next comment.

Reply:
Removed the import!